registered-reports_guide.md

```diff
@@ -0,0 +1,94 @@
+
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+
+* **Example**: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
+
+# Structured Abstract (required)
+The abstract should describe in 200 words or so:
+
+## Background/Context
+What is your research about? Why are you doing this research, why is it interesting?
+
+**Example**: The enterprise is the flag ship of the federation, and it allows families to travel onboard. However, there are no studies that evaluate how this affects the crew members.”
+
+## Objective/Aim
+What exactly are you studying/investigating/evaluating? What are the objects of the study?
+
+We welcome both confirmatory and exploratory types of studies.
+
+**Example**: We evaluate whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family.
+**Example**: We investigate the problem of frequent Holodeck use on interpersonal relationships with an ethnographic study using participant observation, in order to derive specific hypotheses about Holodeck usage.
+
+## Method
+How are you addressing your objective? What data sources are you using.
+
+**Example**: We conduct an observational study and use a between subject design. To analyze the data, we use a t test or Wilcoxon test, depending on the underlying distribution. Our data come computer monitoring of Enterprise crew members.
+
+## Limitations
+
+# Hypotheses / research questions (required)
+Clearly state the research hypotheses that you want to test with your study, and a rationalization for the hypotheses. 
+
+* **Example**: Science officers with their family on board have more sick days than science officers without their family
+
+Rationale: Since toddlers are often sick, we can expect that crew members with their family onboard need to take sick days more often. 
+
+For now, we focus the registered reports track on confirmatory research, ie. where a well-established hypothesis can be formulated. While pre-registration of exploratory studies may be possible, at this time we focus on the confirmatory reports. The NIER track is a good venue for exploratory/emerging results.
+
+# Introduction
+Give more details on the bigger picture of your study and how it contributes to this bigger picture. An important component pf pHase 1 review is assessing the novelty and relevance of the study questions, so be sure to explain this. 
```

Comment:
I would suggest to omit "novelty" here to clearly distinguish from NIER.

Reply:
thanks, I've updated this in the latest patch